airflow/models/dagrun.py

```diff
@@ -132,8 +132,8 @@ class DagRun(Base, LoggingMixin):
     # These two must be either both NULL or both datetime.
     data_interval_start = Column(UtcDateTime)
     data_interval_end = Column(UtcDateTime)
-    # When a scheduler last attempted to schedule TIs for this DagRun
-    last_scheduling_decision = Column(UtcDateTime)
+    # When next a scheduler would attempt to schedule TIs for this DagRun
+    next_schedulable = Column(UtcDateTime, default=timezone.utcnow())
```

Comment:
```suggestion
    next_schedulable = Column(UtcDateTime, default=timezone.utcnow)
```

Pretty sure you want a callable, not the result of when this model is imported.